src/main/java/teammates/logic/core/CoursesLogic.java

```diff
@@ -68,9 +69,7 @@ public static CoursesLogic inst() {
     public void createCourse(String courseId, String courseName, String courseTimeZone)
             throws InvalidParametersException, EntityAlreadyExistsException {
 
-        CourseAttributes courseToAdd = CourseAttributes
-                .builder(courseId, courseName, courseTimeZone)
-                .build();
+        CourseAttributes courseToAdd = validateAndCreateCourseAttributes(courseId, courseName, courseTimeZone);
```

Comment:
Update `testCreateCourse` in `CoursesLogicTest` to test invalid time zone behaviour, now that this is not covered by `CourseAttributesTest`.

Reply:
Ok